### ISSUE TITLE
[Bulk Edit Relates Shows] Edit Multiple Shows from Drop Down

### ISSUE THREAD

**Issue by tonyzeoli** [netmix/radio-station#291]
**Describe the bug**

When i got to bulk edit multiple Posts from the Posts admin by selecting the Posts and then selecting Apply, nothing happens. I can do a Quick Edit of a single Post, but can't assign one or many Posts to one or many Shows using that function of the admin.

**To Reproduce**

See the video: https://www.loom.com/share/0ed48e046993436c9600e7cc2b9bfa8a

**Expected behavior**

Once I select the Posts, I should be able to click Apply and the Shows in a selection box should appear.

**Screenshots**

https://www.loom.com/share/0ed48e046993436c9600e7cc2b9bfa8a

**Comment by majick777** [comment]
@tonyzeoli Seems like a bit of a jQuery conflict. Sometimes it doesn't like `$` being used as the jQuery object. I try to avoid this by writing `jQuery` instead of `$` but failed to in this case. Changing it seems to have fixed the issue on the demo site (it was working for me locally as there was no conflict there.) 

Additionally, I noticed while testing that if you selected the "Set Related Shows" item twice, it added the field twice! So I added another line of code to make sure it is only added once. Added to develop branch for 2.3.3.7 release.
